apps/web/pages/auth/setup/index.tsx

```diff
@@ -1,146 +1,65 @@
 "use client";
 
-import { usePathname, useRouter } from "next/navigation";
+import { useRouter } from "next/navigation";
 import { useState } from "react";
 
-import AdminAppsList from "@calcom/features/apps/AdminAppsList";
-import { APP_NAME } from "@calcom/lib/constants";
-import { useCompatSearchParams } from "@calcom/lib/hooks/useCompatSearchParams";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import type { inferSSRProps } from "@calcom/types/inferSSRProps";
-import { Meta, WizardForm } from "@calcom/ui";
+import { Meta, WizardLayout } from "@calcom/ui";
 
 import PageWrapper from "@components/PageWrapper";
 import { AdminUserContainer as AdminUser } from "@components/setup/AdminUser";
 import ChooseLicense from "@components/setup/ChooseLicense";
-import EnterpriseLicense from "@components/setup/EnterpriseLicense";
 
 import { getServerSideProps } from "@server/lib/setup/getServerSideProps";
 
-function useSetStep() {
-  const router = useRouter();
-  const searchParams = useCompatSearchParams();
-  const pathname = usePathname();
-  const setStep = (newStep = 1) => {
-    const _searchParams = new URLSearchParams(searchParams ?? undefined);
-    _searchParams.set("step", newStep.toString());
-    router.replace(`${pathname}?${_searchParams.toString()}`);
-  };
-  return setStep;
-}
-
 export function Setup(props: inferSSRProps<typeof getServerSideProps>) {
+  const [step, setStep] = useState(1);
   const { t } = useLocale();
   const router = useRouter();
-  const [value, setValue] = useState(props.isFreeLicense ? "FREE" : "EE");
-  const isFreeLicense = value === "FREE";
-  const [isEnabledEE, setIsEnabledEE] = useState(!props.isFreeLicense);
-  const setStep = useSetStep();
 
-  const steps: React.ComponentProps<typeof WizardForm>["steps"] = [
+  const steps = [
     {
       title: t("administrator_user"),
       description: t("lets_create_first_administrator_user"),
-      content: (setIsPending) => (
+      content: () => (
         <AdminUser
-          onSubmit={() => {
-            setIsPending(true);
-          }}
           onSuccess={() => {
             setStep(2);
           }}
-          onError={() => {
-            setIsPending(false);
-          }}
           userCount={props.userCount}
         />
       ),
     },
     {
       title: t("choose_a_license"),
       description: t("choose_license_description"),
-      content: (setIsPending) => {
+      content: () => {
         return (
           <ChooseLicense
-            id="wizard-step-2"
-            name="wizard-step-2"
-            value={value}
-            onChange={setValue}
-            onSubmit={() => {
-              setIsPending(true);
-              setStep(3);
+            licenseStatus={props.licenseStatus}
+            onSuccess={() => {
+              localStorage.setItem("onBoardingRedirect", "/settings/admin/apps");
+              router.replace("/getting-started");
```

Comment:
This is a bit of a hack. 

Making changes to make admin apps card fit this 600px layout wasnt in scope. So we redirect to the page where this was meant to be used.

Reply:
I'm ok redirecting to the admin apps settings page but it feels a little jarring. Are there other actions an admin can take? Can we instead show a "Next steps" page with a link to the apps settings and others? Doesn't have to be in this PR.